classes/class-wc-connect-nux.php

```diff
@@ -1,5 +1,16 @@
 <?php
 
+if ( ! class_exists( 'Jetpack_Install_Status' ) ) {
```

Comment:
@jeffstieler I encapsulated the constants in their own class since they're not directly related to NUX & the Jetpack cxn constants are likely useful outside of NUX (have not run into that yet, though, but I think the first reason stands on its own, too). To me, adding the Jetpack-related constants in this class would seem a bit out of place. Is there another reason to keep them in the NUX class that I'm not thinking of? Maybe simplicity for now? Not tied to either approach atm :)

> I'd go ahead and put this class in its own file.

Sounds good. Will do in a bit (if we keep the class).